lib/components/user/user-account-screen.js

```diff
@@ -0,0 +1,595 @@
+            improve transit services in my area. <a href='/#/terms-of-storage'>More info...</a>
+          </Checkbox>
+        </FormGroup>
+      </div>
+    )
+
+    const notificationPrefsPane = (
+      <div>
+        <p>
+          You can receive notifications about trips you frequently take.
+        </p>
+        <FormGroup>
+          <ControlLabel>How would you like to receive notifications?</ControlLabel>
+          <ButtonToolbar>
+            <ToggleButtonGroup
+              name='notificationChannels'
+              onChange={this._handleNotificationChannelChange}
+              type='checkbox'
+              value={notificationChannels}
+            >
+              {allowedNotificationChannels.map(({ type, text }, index) => (
+                <ToggleButton
+                  bsStyle={notificationChannels.includes(type) ? 'primary' : 'default'}
+                  key={index}
+                  value={type}
+                >
+                  {text}
+                </ToggleButton>
+              ))}
+            </ToggleButtonGroup>
+          </ButtonToolbar>
+        </FormGroup>
+        <div
+          style={{height: '150px'}} // preserve height of pane.
+        >
+          {notificationChannels.includes('email') && (
+            <FormGroup>
+              <ControlLabel>Notification emails will be sent out to:</ControlLabel>
+              <FormControl disabled type='text' value={userData.email} />
+            </FormGroup>
+          )}
+          {notificationChannels.includes('sms') && (
+            <FormGroup>
+              <ControlLabel>Enter your phone number for SMS notifications:</ControlLabel>
+              <FormControl onChange={this._handlePhoneNumberChange} type='tel' value={phoneNumber} />
+            </FormGroup>
+          )}
+        </div>
+      </div>
+    )
+
+    const phoneVerificationPane = (
+      <div>
+        <Alert bsStyle='warning'>
+          <strong>Under construction!</strong>
+        </Alert>
+        <p>
+          Please check your mobile phone's SMS messaging app for a text
+          message with a verification code and copy the code below:
+        </p>
+        <FormGroup bsSize='large'>
+          <FormControl type='number' placeholder='_ _ _ _ _ _' />
+        </FormGroup>
+      </div>
+    )
+
+    // In theory there could be multiple home or work locations.
+    // Just pick the first one,
+    const homeLocation = savedLocations.find(loc => loc.type === 'home') || {
+      address: null,
+      icon: 'home',
+      type: 'home'
+    }
+    const workLocation = savedLocations.find(loc => loc.type === 'work') || {
+      address: null,
+      icon: 'briefcase',
+      type: 'work'
+    }
+
+    // Build an 'effective' locations list for display.
+    const effectiveLocations = [
+      homeLocation,
+      workLocation,
+      ...savedLocations.filter(loc => loc.type !== 'home' && loc.type !== 'work')
+    ]
+
+    const placesPane = (
```

Comment:
Needs a refactor to allow geocoding of locations. Should be in a separate component/file like the other panes.

Reply:
Component has been extracted. Per internal IBI discussion, we will handle geocoding in a separate PR.